### PR TITLE
Functional test for new list flow and some tweaks.

### PR DESCRIPTION
**Pull request by andrewchong**
Task list flow changed. This commit address the changes, add's some missing scenarios, add's some new steps and rejigs some existing steps.
-Search >30 results can't be exported
-Export message in relation to >30 results in saved search
-Check for list item status "Can't start yet', 'Completed' 'Contract awarded to <service name>', 'The work has been cancelled', 'No suitable services found'
-Tweaked export scenario
-Tweaked download scenario
-Removed commented lines that are no longer required

### CODE REVIEW

File: features/step_definitions/direct_award_steps.rb [hunk 1]
```diff
@@ -10,35 +15,23 @@
...
-    And I click the 'Export your results' link
-    Then I am on the 'Before you export your results' page
-    And I check 'I understand that I cannot edit my search again after I export my results' checkbox
-    And I click the 'Export results and continue' button
-    Then I am on the 'my cloud project' page
-    When I have downloaded the search results as a file of type 'ods'
+    Given I have created and saved a search called '#{search_name}'
+    And I am on the '#{search_name}' page
+    And I have exported my results for the '#{search_name}' saved search
     And I click the 'Return to your task list' link
```
Comment (katstevens) [inline features/step_definitions/direct_award_steps.rb]: This line seems to be causing a few failures - clicking the export link goes straight back to the task list (is it meant to do this?).

**Comment by lfdebrux** [comment]
@katstevens Thanks for taking a look at this, I'm taking over this PR as @andrewchong has left DM, so I'll do those tweaks.

Note that these tests are designed for the tasklist stuff that @galund was working on before he left, so they won't work against master until digitalmarketplace-buyer-frontend#809 has been merged.

File: features/buyer/tell_us_about_contract.feature [hunk 2]
```diff
@@ -61,37 +61,73 @@ Scenario: User edits existing search
   And I click 'Save and continue'
   Then I am on the 'my cloud project' page
 
-Scenario: User exports and downloads results
```
Comment (katstevens) [inline features/buyer/tell_us_about_contract.feature]: Let's add `@skip-staging` to these scenarios (the first 5 should be ok).

Comment (lfdebrux) [reply features/buyer/tell_us_about_contract.feature]: Will do

**Pull request merged**